src/main/java/com/codeborne/selenide/impl/AttachmentConsolePrinter.java

```diff
@@ -0,0 +1,17 @@
+package com.codeborne.selenide.impl;
+
+import java.io.File;
+
+public class AttachmentConsolePrinter {
+
+  /**
+   * Prints an attachment line to the console. This is recognised by at least
+   * <a href="https://docs.gitlab.com/ci/testing/unit_test_reports/">GitLab Unit test reports</a>
+   * and <a href="https://plugins.jenkins.io/junit-attachments/">Jenkins JUnit Attachments plugin</a>.
+   *
+   * @param file the file to be attached
+   */
+  public static void printAttachmentLine(File file) {
+    System.out.printf("[[ATTACHMENT|%s]]%n", file.getAbsolutePath());
```

Comment:
@vtintillier Why don't we use slf4j for logging this line?

Like
```java
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

  private static final Logger log = LoggerFactory.getLogger(AttachmentConsolePrinter.class);

  log.info("[[ATTACHMENT|{}]]", file.getAbsolutePath());
```

Reply:
Not sure 😄 

I guess I first wanted to keep it simple and just output the expected line, but I guess gitlab/jenkins don't look for a full line, and just look for a substring in any line. So I can use logger 👍